app/Http/Controllers/MemberController.php

```diff
@@ -12,63 +12,59 @@
 
 class MemberController extends Controller
 {
-    public function listMember(Request $request)
+    public function index(Request $request)
     {
-        $listMember= Member::all()->toArray();
+        $listMember = Member::all() ->toArray();
         return $listMember ;
     }
 
-    public function deleteMember($id)
+    public function destroy(Request $request)
     {
-        $member=Member::find($id);
-        $member ->delete();
-        return "xóa thành công" ;
-    }
-
-    public function getCreateMember()
-    {
-        return view('test');
-    }
-
-    public function getEditMember($id)
-    {
-        $item = Member::find($id)->toArray();
-        return view('editmember', ['id'=>$id, 'item'=>$item]);
+        $id = $request->id;
+        if ($member = Member::find($id)) {
+            $member->delete();
+            $listMember = Member::all() ->toArray();
+            return $listMember ;
+        }
+        return ["message" => "Doesn't Exit Item"];
```

Comment:
Why are you return string here. PLs return a object json :D

Reply:
I have fix